cognite/client/data_classes/workflows.py

```diff
@@ -1385,18 +1385,21 @@ def __init__(
         trigger_fire_time: int,
```

Comment:
Why not also update the data class names? It is better if this matches the spec, and as this feature is in alpha we can do it without a major bump.
```suggestion
        fire_time: int,
```

Reply:
Not sure I follow: why you want to change the data class names?
(the sdk is tagged as beta for now, only a selection of projects have access)

Reply:
If I understand it correctly, this class represent the response object of the getTriggerHistory endpoint. 

Then I expect the attributes to be snake case of the parameters returned in the response. In addition, it seem the `status`and `reasonForFailure` is missing.
![image](https://github.com/user-attachments/assets/5cda5771-2d2c-4e78-956e-03805ed6fc00)